scrapinghub/client.py

```diff
@@ -305,6 +306,16 @@ def __init__(self, client, projectid, spiderid, spidername):
         self.jobs = Jobs(client, projectid, self)
         self._client = client
 
+    @wrap_http_errors
+    def update_tags(self, add=None, remove=None):
+        params = get_tags_for_update(add=add, remove=remove)
+        if not params:
+            return
+        path = 'v2/projects/{}/spiders/{}/tags'.format(self.projectid, self.id)
+        url = urljoin(self._client._connection.url, path)
+        response = self._client._connection._session.patch(url, json=params)
```

Comment:
I don't have any objections to expose it, but I can miss some details here. / cc @michalmo  

If we don't want to expose it for now, we should drop the spider's `list_tags`/`update_tags` from the client, or move the endpoints/add some routers.